template/accordion/accordion-group.html

```diff
@@ -1,4 +1,4 @@
-<div class="panel panel-default">
+<div class="panel panel-{{panelClass || 'default'}}">
```

Comment:
I really went back and forth here on this (as well as putting the default inside the directive ala `scope.panelClass = attrs.panelClass || 'panel-default';` but opted for what I did so as to keep the code as close to the element as possible) but ultimately decided to mimic [what we do in tabs](https://github.com/angular-ui/bootstrap/blob/master/template/tabs/tabset.html#L2).

Recall that we did talk about only supporting the built-in contextual panel classes from Bootstrap - which is also why I went this route.

But I don't have a strong feeling either way and if we want to be totally flexible that solution would do it.